api/v1beta1/hostedcluster_conditions.go

```diff
@@ -106,6 +106,11 @@ const (
 	// A failure here is unlikely to resolve without the changing user input.
 	ValidReleaseImage ConditionType = "ValidReleaseImage"
 
+	// SubOptimalMTUDetected indicates if the MTU configured on an infra cluster
+	// hosting a guest cluster utilizing kubevirt platform is suboptimal and may degrade the performance
+	// due to fragmentation of the double encapsulation in ovn-kubernetes
+	SubOptimalMTUDetected ConditionType = "SubOptimalMTUDetected"
+
```

Comment:
I think we should look at the AWS specific conditions as a precedence for how we name our condition. Since our condition is kubevirt platform specific, maybe we should call it `ValidKubeVirtInfraNetworkMTU`.